helpers/fix_dangling_adaptation-community_repo.sh

```diff
@@ -16,12 +16,12 @@ if [ -n "$(grep '%define community_adaptation' $ANDROID_ROOT/hybris/droid-config
         echo "adaptation-community has been found pointing to an OBS repo, let's purge it for local builds"
         sb2 -t $VENDOR-$DEVICE-$PORT_ARCH -m sdk-install -R rpm -e --nodeps community-adaptation-devel
         BUILDALL=y
-        buildmw https://github.com/mer-hybris/community-adaptation.git rpm/community-adaptation-localbuild.spec || die
+        buildmw -u "https://github.com/mer-hybris/community-adaptation.git" -s rpm/community-adaptation-localbuild.spec || die
```

Comment:
This file is an old relic, and will be removed altogether in a separate commit.

Reply:
Well I don't want to change any real behaviour in this commit. If you push any change I will rebase and incorporate it, but I would like to keep this PR clean.

Reply:
Just pushed to master, let's keep this PR also minimal :)

Reply:
Rebased and deleted.